config/initializers/extends.rb

```diff
@@ -441,11 +441,13 @@ class Extends
     protocol_template_access_granted_all_team_members: 238,
     protocol_template_access_changed_all_team_members: 239,
     protocol_template_access_revoked_all_team_members: 240,
-    project_access_changed_all_team_members: 241
+    project_access_changed_all_team_members: 241,
+    project_grant_access_to_all_team_members: 242,
+    project_remove_access_from_all_team_members: 243
   }
 
   ACTIVITY_GROUPS = {
-    projects: [*0..7, 32, 33, 34, 95, 108, 65, 109, *158..162, 241],
+    projects: [0, 1, *3..7, 32, 33, 34, 95, 108, 65, 109, *158..162, 241, 242, 243],
```

Comment:
why did you remove 2?

Reply:
Because we need to 
> Remove the activity type “Project visibility changed” 
` ACTIVITY_GROUPS` is only being used to generate filters for activities.